05.wc/wcc.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+WIDTH = 8
+
+def main(args)
+  options = {}
+  options[:filepaths] = args[:filepaths] || args[:filepaths] = false
+  options[:l]         = args[:l] || args[:l] = false
+  options[:w]         = args[:w] || args[:w] = false
+  options[:c]         = args[:c] || args[:c] = false
+
+  if options[:filepaths].empty?
+    show_stats_pipe_input(options)
+  else
+    show_stats_file_input(options)
+  end
+end
+
+def show_stats_pipe_input(options)
```

Comment:
```suggestion
def show_stats_for_pipe_input(options)
```

何かしら前置詞を付けた方が英語っぽく読めそうです

Reply:
承知しました。ご指摘のとおり修正しました